enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,176 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@wking"
+reviewers:
+  - "@abhinavdahiya"
+  - "@crawford"
+  - "@smarterclayton"
+approvers:
+  - TBD
+creation-date: 2019-11-19
+last-updated: 2019-11-21
+status: implementable
```

Comment:
possibly update this status to `deferred` given the frozen state if we want to merge for record keeping?